src/Fields/DateRangePickerField.tsx

```diff
@@ -33,7 +33,11 @@ export function DateRangePickerField<
   } = useController<TFieldValues, TName>(controller);
   return (
     <FieldWrapper controller={controller} formItem={formItem}>
-      <RangePicker format="DD.MM.YYYY" {...fieldProps} {...component} />
+      <RangePicker
+        format={['DD.MM.YYYY', 'DDMMYYYY']}
```

Comment:
What does this change accomplish?

Reply:
its adds 'DDMMYYYY' as possible format, but wait there is another change related to the rangepicker incoming

Reply:
I don't understand how this impacts how this component works. I assume it only concerns data entry, but the component still canonicalizes the values it stores internally?

Reply:
"When an array is provided, all values are used for parsing and first value is used for formatting" https://4x.ant.design/components/date-picker/#API

Reply:
Nice, thank you!